app/assets/javascripts/custom.js

```diff
@@ -52,15 +82,17 @@ $(document).ready(function(){
         },
         order_details: {
           sub_total: line_total,
-          total: _total
+          total: _total,
+          prep_time: line_prep_total,
+          prep_total: _preptotal
         }
       };
     var url = "/cart_items/1";
     ajax_call($(this), params, url, "PATCH", function(data){
-        $('#food_sub_total_' + food_id).text(line_total);
         $("#total").text("N" + _total.toFixed(2));
+        $("#prep_total").text("Your order will be ready in: " + _preptotal + "mins");
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Too many errors. (57% scanned).